beam/README.md

```diff
@@ -8,12 +8,9 @@ Current interpreter implementation supports the static repl. It compiles the cod
 You have to first build the Beam interpreter by enable the **beam** profile as follows:
 
 ```
-mvn clean package -Pbeam -DskipTests
+mvn clean package -Pbeam -DskipTests -Pscala-2.10
```

Comment:
It's still `scala-2.10` only in Beam 2.0, let me restore the notice message. --Btw, does Zeppelin provide a release version with `scala-2.11`(donot find in download page)? Wanna know what will happen with Beam.